enos/enos-scenario-package-manager.hcl

```diff
@@ -0,0 +1,217 @@
+scenario "package_manager" {
+  matrix {
+    arch            = ["amd64", "arm64"]
+    backend         = ["consul", "raft"]
+    artifact_source = ["local", "crt", "artifactory"]
+    consul_version  = ["1.13.2"]
```

Comment:
Yeah, it's an odd thing for a matrix but I left it this way so that the matrix and the rest of the scenario maintain a similar structure as the other enos scenarios (smoke, upgrade) which do test across three different consul versions.

Reply:
Yeah, I agree with mike. For single variants it's probably best to use a variable with a default so you don't have to write out the consul version when you're filtering.